### PR TITLE
MAP-171-Move end points over to use new API in reservations

### PR DESCRIPTION
**Pull request by ratheeshri1984**

### CODE REVIEW

File: src/Reservations/SFA.DAS.Reservations/Application/Providers/Queries/GetProvider/GetProviderQueryHandler.cs [hunk 1]
```diff
@@ -11,23 +11,44 @@ namespace SFA.DAS.Reservations.Application.Providers.Queries.GetProvider
...
         }
 
         public async Task<GetProviderResult> Handle(GetProviderQuery request, CancellationToken cancellationToken)
         {
-            var provider = await _courseDeliveryApiClient.Get<GetProviderResponse>(
-                new GetProviderRequest
-                {
-                    Ukprn = request.Ukprn
-                });
+            GetProviderResponse providers=null;
```
Comment (cofaulco) [inline src/Reservations/SFA.DAS.Reservations/Application/Providers/Queries/GetProvider/GetProviderQueryHandler.cs:28]: singular `provider` for variable name

File: src/Reservations/SFA.DAS.Reservations/Application/Providers/Queries/GetProvider/GetProviderQueryHandler.cs [hunk 2]
```diff
@@ -11,23 +11,44 @@ namespace SFA.DAS.Reservations.Application.Providers.Queries.GetProvider
...
+
+            if (_featureToggles.RoatpProvidersEnabled)
+            {
+                var provider = await _roatpApiClient.Get<GetRoatpProviderResponseSummary>(
```
Comment (cofaulco) [inline src/Reservations/SFA.DAS.Reservations/Application/Providers/Queries/GetProvider/GetProviderQueryHandler.cs:32]: rename to `roatpProviderResponse`

**Review by cofaulco — CHANGES_REQUESTED** [review]
couple of suggestions

File: src/Reservations/SFA.DAS.Reservations/Application/Providers/Queries/GetProvider/GetProviderQueryHandler.cs (hunk 1 above)
Comment (ratheeshri1984) [reply src/Reservations/SFA.DAS.Reservations/Application/Providers/Queries/GetProvider/GetProviderQueryHandler.cs]: Done

File: src/Reservations/SFA.DAS.Reservations/Application/Providers/Queries/GetProvider/GetProviderQueryHandler.cs (hunk 2 above)
Comment (ratheeshri1984) [reply src/Reservations/SFA.DAS.Reservations/Application/Providers/Queries/GetProvider/GetProviderQueryHandler.cs]: No change

**Review by cofaulco — APPROVED** [review]

**Review by cofaulco — APPROVED** [review]

**Pull request merged**